### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [snapappointments/bootstrap-select#2810] [Q&A] [answered]
Hi.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by NicolasCARPi** [comment, marked as answer]
See my message here: https://github.com/snapappointments/bootstrap-select/issues/2228#issuecomment-1236207188

**Reply by ghost** [reply]
Hi NicolasCARPi.

> See my message here: https://github.com/snapappointments/bootstrap-select/issues/2228#issuecomment-1236207188

thank you for feedback.

**Reply by cHillb3rt** [reply]
And what could maintenaned from another people in this project?
